Update method for retrieving 'oauth_consumer_key' and 'oauth_consumer…

illumino opened the pull request:
…_secret'

https://projects.theforeman.org/issues/31472
https://community.theforeman.org/t/web-site-manual-2-3-0-out-of-date-instructions-on-obtaining-oauth-consumer-key-and-oauth-consumer-secret/21586

illumino committed:
Update method for retrieving 'oauth_consumer_key' and 'oauth_consumer_secret'

https://projects.theforeman.org/issues/31472
https://community.theforeman.org/t/web-site-manual-2-3-0-out-of-date-instructions-on-obtaining-oauth-consumer-key-and-oauth-consumer-secret/21586

ekohl reviewed (changes requested):
I would still prefer to get them via `foreman-rake config | grep oauth` or `foreman-rake -- config -k oauth_consumer_key`. Those are official interfaces that I would support. They use the application logic. When you don't use the installer, you can leave them out of `settings.yaml` and only set them in the database. Following the application logic prepares us better for a future where we might want to support containers.

illumino committed:
Update method for retrieving 'oauth_consumer_key' and 'oauth_consumer… e79d2bb  …_secret' (Revised)

As requested, updated to use foreman-rake.
Note: used 'sudo', as I was not able to determine the password an unprivelged user should use when executing the command without sudo(8)

illumino commented:
@ekohl did the update not meet your requirements?
Or did the changes just get missed?

ekohl commented:
>  Or did the changes just get missed?

This. My apologies for keeping this open so long.
